### PR TITLE
Split Redis docs

### LINKED ISSUE #11130: Split the Redis Database Access guide
## Details
As [discussed](https://github.com/gravitational/teleport/pull/10693#discussion_r824142303) in the PR for our Redis Database Access guide, the guide would be easier to follow if split in two, with one page for Redis Standalone and the other for Redis Cluster. We didn't have time to make this change before the v9 release, so I am submitting this issue so we can schedule this work for later.

### Category
- Improve Existing

### PR DESCRIPTION
**Pull request by jakule**
Split Redis documentation to Redis Standalone and Redis Cluster. There are many differences in configuration and current version of the docs has many "Only Redis Cluster" blocks, that makes it hard to follow.
I've also changed some formatting.

Closes #11130

### CODE REVIEW

File: docs/pages/database-access/guides/redis-cluster.mdx [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+
+(!docs/pages/includes/database-access/create-user.mdx!)
+
+## Step 3/6. Create Redis users
+
+(!docs/pages/includes/database-access/redis-create-users.mdx!)
+
+## Step 4/6. Set up mutual TLS
+
+(!docs/pages/includes/database-access/tctl-auth-sign.mdx!)
```
Comment (ptgott) [inline docs/pages/database-access/guides/redis-cluster.mdx:48]: I think there should be a clearer transition between the use of the `tctl-auth-sign.mdx` partial and the `### Create the secrets` section. Added a suggestion:

```suggestion
(!docs/pages/includes/database-access/tctl-auth-sign.mdx!)

We will show you how to use the `tctl auth sign` command below.
```

File: docs/pages/database-access/guides/redis-cluster.mdx [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+
+### Create the secrets
```
Comment (ptgott) [inline docs/pages/database-access/guides/redis-cluster.mdx:50]: I think we can remove this heading since it's the only H3-level heading in the "Set up mutual TLS" section.

```suggestion
```

File: docs/pages/includes/database-access/redis-connect.mdx [hunk 3]
```diff
@@ -0,0 +1,70 @@
+Log into your Teleport cluster and see available databases:
+
+<Tabs>
+  <TabItem scope={["enterprise", "oss"]} label="Self Hosted">
```
Comment (ptgott) [inline docs/pages/includes/database-access/redis-connect.mdx:4]:
```suggestion
  <TabItem scope={["enterprise", "oss"]} label="Self-Hosted">
```
to be consistent with other uses of this tab label in our docs

File: docs/pages/includes/database-access/redis-connect.mdx [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    ```code
+    $ tsh login --proxy=teleport.example.com --user=alice
+    $ tsh db ls
+    # Name          Description     Labels
+    # ------------- --------------- --------
+    # example-redis Example Redis   env=dev
+    ```
+  </TabItem>
+  <TabItem scope={["cloud"]} label="Cloud">
```
Comment (ptgott) [inline docs/pages/includes/database-access/redis-connect.mdx:13]:
```suggestion
  <TabItem scope={["cloud"]} label="Teleport Cloud">
```

File: docs/pages/includes/database-access/redis-connect.mdx [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+</Tabs>
+
+To connect to a particular database instance, first retrieve its certificate
+using the `tsh db login` command:
+
+```code
+$ tsh db login example-redis
+```
+
+<Admonition type="tip" title="Tip">
```
Comment (ptgott) [inline docs/pages/includes/database-access/redis-connect.mdx:31]: I suggest we remove the `Admonition` tag and make this regular text. I'm not sure how likely it is that a user will miss this after copying the preceding code snippet. Otherwise, this section looks really busy.

File: docs/pages/includes/database-access/redis-connect.mdx [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+</Admonition>
+
+You can optionally specify the database name and the user to use by default
+when connecting to the database instance:
+
+```code
+$ tsh db login --db-user=alice example-redis
+```
+
+<Admonition type="note" title="Note">
```
Comment (ptgott) [inline docs/pages/includes/database-access/redis-connect.mdx:42]: Same as before re: removing the `Admonition` tag

File: docs/pages/includes/database-access/redis-connect.mdx [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+  If flag `--db-user` is not provided, Teleport logs in as the `default` user.
+</Admonition>
+
+Once logged in, connect to the database:
+
+```code
+$ tsh db connect example-redis
+```
+
+<Admonition type="note" title="Note">
```
Comment (ptgott) [inline docs/pages/includes/database-access/redis-connect.mdx:52]: Same as before re: the `Admonition` tag

File: docs/pages/includes/database-access/redis-connect.mdx [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+  The `redis-cli` command-line client should be available in the system `PATH` in order to be
+  able to connect.
+</Admonition>
+
+Now you can log in as the previously created user using the below command:
+
+```
+AUTH alice STRONG_GENERATED_PASSWORD
```
Comment (ptgott) [inline docs/pages/includes/database-access/redis-connect.mdx:60]: Was the `AUTH` variable assigned to something before?

File: docs/pages/database-access/guides/redis-cluster.mdx [hunk 9]
```diff
@@ -0,0 +1,185 @@
...
+- `SLAVEOF`
+- `SLOWLOG`
+- `SSUBSCRIBE`
+- `SUNSUBSCRIBE`
+- `SYNC`
+- `TIME`
+- `WAIT`
+- `WATCH`
+
+Teleport also adds additional processing to:
```
Comment (ptgott) [inline docs/pages/database-access/guides/redis-cluster.mdx:170]:
```suggestion
Teleport conducts additional processing on the following commands before 
communicating with Redis Cluster:
```
or something more specific/less repetitive than "adds additional"

File: docs/pages/database-access/guides/redis.mdx [hunk 10]
```diff
@@ -13,6 +13,8 @@ description: How to configure Teleport Database Access with Redis.
   Database access for Redis is available starting from Teleport `9.0`.
 </Details>
 
+If you want to configure Redis Cluster guide is available [here](redis-cluster.mdx).
```
Comment (ptgott) [inline docs/pages/database-access/guides/redis.mdx:16]:
```suggestion
If you want to configure Redis Cluster, please read [Database Access with Redis Cluster](redis-cluster.mdx).
```

File: docs/pages/database-access/guides/redis.mdx [hunk 11]
```diff
@@ -39,146 +41,47 @@ This guide will help you to:
...
-    $ tctl auth sign --format=redis --host=redis.example.com --out=server --ttl=2190h
-    ```
-
-    (!docs/pages/includes/database-access/ttl-note.mdx!)
-
-    The command will create three files:
-    - `server.cas` with Teleport's certificate authority
-    - `server.key` with a generated private key
-    - `server.crt` with a generated user certificate
+### Create the secrets
```
Comment (ptgott) [inline docs/pages/database-access/guides/redis.mdx:50]:
```suggestion
We will show you how to use `tctl auth sign` below.
```
As suggested with the other Redis guide

File: docs/pages/includes/database-access/guides.mdx [hunk 12]
```diff
@@ -34,6 +34,9 @@
   <Tile icon="database" title="Self-hosted Redis" href="./guides/redis.mdx">
     Connect self-hosted Redis.
   </Tile>
+  <Tile icon="database" title="Self-hosted Redis Cluster" href="./guides/redis-cluster.mdx">
+    Connect self-hosted Redis Cluster.
```
Comment (ptgott) [inline docs/pages/includes/database-access/guides.mdx:38]:
```suggestion
    Connect a self-hosted Redis Cluster.
```

**Review by ptgott — COMMENTED** [review]
Thanks for putting this together!

File: docs/pages/database-access/guides/redis-cluster.mdx [hunk 13]
```diff
@@ -0,0 +1,185 @@
...
+    uri: "rediss://redis.example.com:6379?mode=cluster"
+```
+
+(!docs/pages/includes/database-access/redis-connect.mdx!)
+
+### Supported Redis Cluster commands
+
+Redis in cluster mode doesn't support below commands. If one of the listed commands above is called Teleport
+returns the <nobr>`ERR Teleport: command not supported`</nobr> error.
+- `ACL`
```
Comment (r0mant) [inline docs/pages/database-access/guides/redis-cluster.mdx:131]: Can this long list be hidden under collapsible section? I think we have a "details" component for this.

**Review by r0mant — APPROVED** [review]

File: docs/pages/includes/database-access/redis-connect.mdx (hunk 5 above)
Comment (jakule) [reply docs/pages/includes/database-access/redis-connect.mdx:31]: I can remove it, but we use that tip in may different places and every elsewhere is inside `Admonition`

File: docs/pages/includes/database-access/redis-connect.mdx (hunk 6 above)
Comment (jakule) [reply docs/pages/includes/database-access/redis-connect.mdx:42]: Removed

File: docs/pages/includes/database-access/redis-connect.mdx (hunk 7 above)
Comment (jakule) [reply docs/pages/includes/database-access/redis-connect.mdx:52]: Removed

File: docs/pages/database-access/guides/redis-cluster.mdx [hunk 14]
```diff
@@ -0,0 +1,190 @@
...
+databases:
+  - name: "redis-cluster"
+    uri: "rediss://redis.example.com:6379?mode=cluster"
+```
+
+(!docs/pages/includes/database-access/redis-connect.mdx!)
+
+### Supported Redis Cluster commands
+
+Redis in cluster mode doesn't support below commands. If one of the listed commands above is called Teleport
```
Comment (ptgott) [inline docs/pages/database-access/guides/redis-cluster.mdx:130]:
```suggestion
Redis in cluster mode does not support the following commands. If one of the listed commands above is called Teleport
```

**Review by ptgott — APPROVED** [review]
One minor nit
